server/service/src/invoice_line/mod.rs

```diff
@@ -53,12 +53,12 @@ pub trait InvoiceLineServiceTrait: Sync + Send {
         insert_stock_out_line(ctx, input)
     }
 
-    fn update_outbound_shipment_line(
+    fn update_stock_out_line(
```

Comment:
I think here and for `insert_stock_out_line` you could add a comment that stock out could mean outbound shipment or prescription. Or is there a better place to put this information?

Reply:
Will do it in #1989, so can put the comment in insert/update/delete at once (: